api/src/main/java/com/nicknathanjustin/streamercontracts/twitch/TwitchService.java

```diff
@@ -0,0 +1,14 @@
+package com.nicknathanjustin.streamercontracts.twitch;
+
+import java.util.List;
+import java.util.Map;
+
+public interface TwitchService {
+
+    /**
+     * Get the top games from the twitch api
+     *
+     * @return A List of top games
```

Comment:
Does it? It looks like its returning a map of strings that map to a list of mapped object. Lets update the return to callout what each of the maps contain